File: go/docmost/models/offset_pagination_meta.go
package models

// OffsetPaginationMeta from OpenAPI.
type OffsetPaginationMeta struct {
	Limit *int `json:"limit,omitempty"`
	Page *int `json:"page,omitempty"`
	HasNextPage *bool `json:"hasNextPage,omitempty"`
	HasPrevPage *bool `json:"hasPrevPage,omitempty"`
}

File: c#/src/Docmost.Sdk/Models/OffsetPaginationMeta.cs
namespace Docmost.Sdk.Models;

public class OffsetPaginationMeta
{
    [System.Text.Json.Serialization.JsonPropertyName("limit")]
    public int? Limit { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("page")]
    public int? Page { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("hasNextPage")]
    public bool? HasNextPage { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("hasPrevPage")]
    public bool? HasPrevPage { get; set; }

}
